masterlist.yaml

```
@@ -2011,47 +2011,88 @@ plugins:
       - crc: 0x165C9A2F
         util: 'SSEEdit v4.0.3'
 
-  - name: 'dD - Enhanced Blood Main.*\.esp'
+  - name: '(dD.*(Blood|Drips|Enhanced Blood Main.*|Range|Size)|EBT - IC PATCH)\.esp'
```

The regex can be made a bit less greedy without sacrificing readability:

```suggestion
  - name: '(dD.*(Blood|Drips|Enhanced Blood Main( LITE)?|Range|Size)|EBT - IC PATCH)\.esp'
```